lms/djangoapps/certificates/api.py

```diff
@@ -211,10 +212,14 @@ def has_html_certificates_enabled(course_key, course=None):
     It determines if course has html certificates enabled
     """
     html_certificates_enabled = False
-    if settings.FEATURES.get('CERTIFICATES_HTML_VIEW', False):
+    try:
+        if not isinstance(course_key, CourseKey):
+            course_key = CourseKey.from_string(course_key)
         course = course if course else modulestore().get_course(course_key, depth=0)
-        if get_active_web_certificate(course) is not None:
```

Comment:
active configuration check does not make sense in this function(`has_html_certificates_enabled`) since it determines whether we have html certs are enabled or not.